src/main/java/ru/skypro/flea/entity/Comment.java

```diff
@@ -0,0 +1,39 @@
+package ru.skypro.flea.entity;
+
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import ru.skypro.flea.converter.LocalDateTimeConverter;
+
+import javax.persistence.*;
+import javax.validation.constraints.NotNull;
+import java.time.LocalDateTime;
+
+@Entity
+@Table(name = "comments")
+@NoArgsConstructor
+@Getter
+@Setter
+public class Comment {
+  @Id
+  @GeneratedValue(strategy = GenerationType.IDENTITY)
+  @Column(name = "id", nullable = false)
+  private Integer id;
+
+  @Column(name = "text", nullable = false)
+  private String title;
+
+  @Column(name = "pubic_date")
+  @Convert(converter = LocalDateTimeConverter.class)
+  @NotNull
```

Comment:
i guess there is no need to validate entities as we use DTOs to communicate with client
so, @NotNull seems to be redundant
at least right now...